tests/python/gpu/test_forward.py

```diff
@@ -1,61 +0,0 @@
-import os
```

Comment:
that's true. but this test is somewhat duplicated with https://github.com/dmlc/mxnet/blob/master/example/image-classification/test_score.py